Bottom: e9d1fa5b746f5d40ed2e08db7be1d6ebdaa6c711
Top:    91c9e13dc1119b42e57be188dcead3c13c2937c1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-04 15:39:47 +0000

gtkhid: Return NULL from ghid_get_net_from_node_name()

Fixes calling return; from function returning non-void type.



---

diff --git a/src/hid/gtk/gui-netlist-window.c b/src/hid/gtk/gui-netlist-window.c
index fe17b31..39ffb4e 100644
--- a/src/hid/gtk/gui-netlist-window.c
+++ b/src/hid/gtk/gui-netlist-window.c
@@ -865,7 +865,7 @@ ghid_get_net_from_node_name (gchar * node_name, gboolean enabled_only)
 
   /* If no netlist is loaded the window doesn't appear. */
   if (netlist_window == NULL)
-    return;
+    return NULL;
 
   task.enabled_only = enabled_only;
   task.node_name = node_name;
